hcfb-fe/src/app/rescue/rescue.service.ts

```diff
@@ -11,11 +19,21 @@ export class RescueService {
   });
 
   currentRescue = this.rescueSource.asObservable();
-  constructor() { }
-  
+  constructor(private http: HttpClient ) { }
+  options = {headers : {'Content-Type' : 'application/json'}};
+
+  private static _handleError(err: HttpErrorResponse | any) {
+    return throwError(err.message || 'Error: Unable to complete request.');
+ }
+
   changeRescue(rescue: Rescue) {
     this.rescueSource.next(rescue)
+  }
 
+  getRescueLocation(): Observable<any> {
+    return this.http
+    .get("http://localhost:8080/api/rescues/1")
```

Comment:
Recommendation: let's extract URL construction into its own object. Then we can tackle pointing this at the real backend in production, and making it configurable for local development, without having to find-replace all over the place.